### DISCUSSION TITLE
v-menu slot default

### DISCUSSION THREAD

**Discussion by carloschneider** [vuetifyjs/vuetify#17477] [Q&A] [answered]
Hello guys! I'm analyzing the source code of Vuetify v2.x `v-menu` component and I was wondering in which part of the code the slot default is sent to outside of the `v-menu` content? To be more specific the `v-menu` content is appended to `v-application`, but I'm trying to discover which part of the code does this.

**Comment by KaelWD** [comment, marked as answer]
https://github.com/vuetifyjs/vuetify/blob/ffbaae129d36e40bb2926914f2960b09370befb2/packages/vuetify/src/mixins/detachable/index.ts

**Comment by carloschneider** [comment]
@KaelWD Thank you, bro! It really helps me! But now I'm trying to find which part of the code is responsible for hide and show (set display none) the content element. You know where this magic happen?

**Comment by KaelWD** [comment]
https://github.com/vuetifyjs/vuetify/blob/ffbaae129d36e40bb2926914f2960b09370befb2/packages/vuetify/src/components/VMenu/VMenu.ts#L322
